src/containers/FormikForm/GrepCodesField.tsx

```diff
@@ -6,20 +6,209 @@
  *
  */
 
-import { FieldProps, FormikValues } from "formik";
-import { memo } from "react";
+import { useField } from "formik";
+import difference from "lodash/difference";
+import uniq from "lodash/uniq";
+import { memo, useState, useEffect, useMemo } from "react";
 import { useTranslation } from "react-i18next";
-import GrepCodesFieldContent from "./GrepCodesFieldContent";
-import FormikField from "../../components/FormikField";
+import { createListCollection, useTagsInputContext } from "@ark-ui/react";
+import { CloseLine } from "@ndla/icons/action";
+import { ArrowDownShortLine } from "@ndla/icons/common";
+import { CheckLine } from "@ndla/icons/editor";
+import {
+  ComboboxItem,
+  ComboboxItemIndicator,
+  ComboboxItemText,
+  FieldHelper,
+  FieldLabel,
+  FieldRoot,
+  IconButton,
+  Input,
+  InputContainer,
+  TagsInputItem,
+  TagsInputItemDeleteTrigger,
+  TagsInputItemInput,
+  TagsInputItemPreview,
+  TagsInputItemText,
+  Text,
+} from "@ndla/primitives";
+import { Flex, HStack, Wrap } from "@ndla/styled-system/jsx";
+import {
+  TagSelectorClearTrigger,
+  TagSelectorControl,
+  TagSelectorInputBase,
+  TagSelectorLabel,
+  TagSelectorRoot,
+  TagSelectorTrigger,
+  useTagSelectorTranslations,
+} from "@ndla/ui";
+import { SearchTagsContent } from "../../components/Form/SearchTagsContent";
+import { FormField } from "../../components/FormField";
+import { useGrepCodesSearch } from "../../modules/draft/draftQueries";
+import { fetchGrepCodeTitle } from "../../modules/grep/grepApi";
+import { GrepCode } from "../../modules/grep/grepApiInterfaces";
+import { isGrepCodeValid } from "../../util/articleUtil";
+import { usePaginatedQuery } from "../../util/usePaginatedQuery";
+
+export const convertGrepCodesToObject = async (grepCodes: string[]): Promise<Record<string, string>> => {
+  const grepCodesWithTitle = await Promise.all(
+    grepCodes.map(async (c) => {
+      const grepCodeTitle = await fetchGrepCodeTitle(c);
+      return {
+        [c]: grepCodeTitle ? `${c} - ${grepCodeTitle}` : c,
+      };
+    }),
+  );
+  return Object.assign({}, ...grepCodesWithTitle);
+};
+
+interface GrepCodeTagItemsProps {
+  grepCodes: Record<string, string>;
+}
+const GrepCodeTagItems = ({ grepCodes }: GrepCodeTagItemsProps) => {
+  const tagsApi = useTagsInputContext();
+  return (
+    <Wrap gap="3xsmall">
+      {tagsApi.value.map((value, index) => (
+        <TagsInputItem index={index} value={value} key={value}>
+          <TagsInputItemPreview>
+            <TagsInputItemText>{grepCodes[value] ?? value}</TagsInputItemText>
+            <TagsInputItemDeleteTrigger>
+              <CloseLine />
+            </TagsInputItemDeleteTrigger>
+          </TagsInputItemPreview>
+          <TagsInputItemInput />
+        </TagsInputItem>
+      ))}
+    </Wrap>
+  );
+};
```

Comment:
Hvorfor må vi gjøre dette?

Reply:
For å få valgte verdier til å legge seg under input, titlene til grep-koder er for lange til å ha inne i inputen, så da ble det å beholde de under

Reply:
Vil da tro at det er greiere å aligne oss på samme måte som vi har gjort andre steder vi bruker multi-combobox, altså at vi viser en fullverdig liste under combobox.